pkg/portworx/cops.go

```diff
@@ -0,0 +1,48 @@
+/*
```

Comment:
This file and the k8s.go file can probably be combined and be moved to `pkg/kubernetes` since it is I didn't see any Portworx specific code there, right?

Reply:
Done. Will be part of next checkin